persistence/soft-index/src/main/java/org/infinispan/persistence/sifs/SoftIndexFileStore.java

```diff
@@ -443,6 +443,9 @@ private ByteBuffer toBuffer(byte[] array) {
    }
 
    private void forEachOnDisk(boolean readMetadata, boolean readValues, EntryFunctor functor, IntConsumer fileFunctor) throws PersistenceException {
+      if (!started) {
```

Comment:
I  have a feeling this one isn't going to be saved either as it nulls out all the values.